src/App.js

```diff
@@ -1,19 +1,22 @@
-import React from 'react';
-
+import React, {useState} from 'react';
 import AddTodoForm from './AddTodoForm';
 import TodoList from './TodoList';
 
 function App(){
 
-  const [ newTodo, setNewTodo ] = React.useState(' ');
+  const [todoList, setTodoList] = useState([]);
+
+  const addTodo= (newTodo) => {
+    setTodoList([...todoList, {newTodo}])
```

Comment:
Cool! You can remove {} for newTodo obj
`setTodoList([...todoList, newTodo])`

Reply:
Hi! If I remove the {} from the newTodo obj it breaks the app.